### ISSUE TITLE
Impossible to use v3.3.0 as AMD after r.js optimization

### ISSUE THREAD

**Issue by mokkabonna** [openlayers/ol3#3389]
It is releated to this issue: https://github.com/mourner/rbush/pull/29

v3.3.0 depends on 1.3.4 of rbush that causes a problem when using ol3 as an AMD module.

Could updating to 1.3.5 be done as a patch version release of openlayers?

This will allow optimizers (r.js) to output this:
```javascript
(function (root, factory) {
  if (typeof define === "function" && define.amd) {
    define('ol',[], factory); //<= optimizer inserts name of module 
  } else if (typeof exports === "object") {
    module.exports = factory();
  } else {
    root.ol = factory();
  }
}(this, function () {
  var OPENLAYERS = {};
  //not relevant code snipped
  if("function"===typeof define && define.amd) define('rbush',[],function (){return {};});
  //not relevant code snipped

  return OPENLAYERS.ol;
}));
```

Instead of what it currently does, this:
```javascript
(function (root, factory) {
  if (typeof define === "function" && define.amd) {
    define([], factory); //<= optimizer trips totally in this case and does nothing, then below creates ol as an empty function 
  } else if (typeof exports === "object") {
    module.exports = factory();
  } else {
    root.ol = factory();
  }
}(this, function () {
  var OPENLAYERS = {};
  //not relevant code snipped
  if("function"===typeof define && define.amd) define(function (){return {};});
  //not relevant code snipped

  return OPENLAYERS.ol;
}));

define("ol", function(){});
```

That obviously renders ol useless..

**Comment by ahocevar** [comment]
OpenLayers 3.4.0 will be released next week. Will that be early enough?

**Comment by mokkabonna** [comment]
Ah great. The fix will be in that release? Good enugh for me.

**Comment by ahocevar** [comment]
> Ah great. The fix will be in that release? Good enugh for me.

Yes, because the issue was fixed in master with #3335 already.
